src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php

```diff
@@ -44,6 +44,33 @@ public function testSubmitEmpty()
         $this->assertNull($form->getData());
     }
 
+    public function testSubmitEmptyMultiple()
+    {
+        $form = $this->factory->createBuilder('file', null, array(
+            'multiple' => true,
+        ))->getForm();
+
+        // submitted data when an input file is uploaded without choosing any file
```

Comment:
This is often the case, for example, when editing a form requesting to change the related image optionally.